exercises/module6/6.2/requests.http

```diff
@@ -0,0 +1,75 @@
+@hostname = localhost
+@port = 3001
+@host = http://{{hostname}}:{{port}}
+@personId = 3
```

Comment:
The API works with UUID, not with integer. Therefore this script might be misleading...